doc/source/extending-jupyterhub.rst

```diff
@@ -198,26 +198,58 @@ your users & the world at large. Zero to JupyterHub makes doing so quite
 easy since version 0.5, integrating with `Let's Encrypt <https://letsencrypt.org/>`_
 for free HTTPS certificates.
 
+You can also purchase your own SSL certificates from a certificate provider.
+
 1. Buy a domain name from a registrar. Pick whichever one you want.
 2. Create an ``A record`` from the domain you want to use, pointing to the
    external IP provided to the `proxy-public` service.
```

Comment:
could explicitly say this is the `<EXTERNAL-IP>` field. For whatever reason people almost always get this confused and copypaste the internal IP

Reply:
(I know this isn't related to this PR so we can punt it to later if you like)